code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/PendingIntentUtils.kt

```diff
@@ -104,9 +105,13 @@ internal object PendingIntentUtils {
         trackerActivityClass: Class<out Activity>?,
         actionUri: String?,
         actionID: String?,
+        actionType: PushTemplateConstants.ActionType?,
         intentExtras: Bundle?
     ): PendingIntent? {
-        val intent = Intent(PushTemplateConstants.NotificationAction.CLICKED)
+        val intent = if (PushTemplateConstants.ActionType.DISMISS == actionType) {
+            Intent(PushTemplateConstants.NotificationAction.DISMISSED)
+        } else Intent(PushTemplateConstants.NotificationAction.CLICKED)
+
```

Comment:
nit: Wondering if we should call these `CLICK` and `DISMISS` since the notification isn't technically dismissed yet and the caller extension needs to do that

Reply:
i agree that this would make sense. i'll update the names.

Reply:
thinking about the naming a bit further, this name is being assigned to a pending intent which the receiving tracker activity is handling. in the tracker activity the name makes sense there as the pending intent is signaling that a clicked or dismissed event occurred. i think we should keep the name as is. what do you think?